ably/options_test.go

```diff
@@ -2,11 +2,195 @@ package ably_test
 
 import (
 	"net/url"
+	"reflect"
 	"testing"
 
 	"github.com/ably/ably-go/ably"
+	"github.com/stretchr/testify/assert"
 )
 
+func TestDefaultFallbacks_RSC15h(t *testing.T) {
+	t.Parallel()
+	t.Run("with env should return environment fallback hosts", func(ts *testing.T) {
+		expectedFallBackHosts := []string{
+			"a.ably-realtime.com",
+			"b.ably-realtime.com",
+			"c.ably-realtime.com",
+			"d.ably-realtime.com",
+			"e.ably-realtime.com",
+		}
+		hosts := ably.DefaultFallbackHosts()
+		assert.True(ts, reflect.DeepEqual(expectedFallBackHosts, hosts), " %s should match %s", hosts, expectedFallBackHosts)
+	})
+}
+
+func TestEnvFallbackHosts_RSC15i(t *testing.T) {
+	t.Parallel()
+	t.Run("with env should return environment fallback hosts", func(ts *testing.T) {
```

Comment:
Naming the inner `*testing.T` also `t` is common practice and ensures that we don't use the external `t` accidentally.